cyclus_input/predicting_the_past.xml

```diff
@@ -170,10 +170,10 @@
       <name>Power</name>
       <config>
         <DeployInst>
-          <xi:include href="./cyclus_input/recipes/lifetimes.xml#xpointer(/simulation/DeployInsts/child::*)" />
+          <xi:include href="./recipes/lifetimes.xml#xpointer(/simulation/DeployInsts/child::*)" />
```

Comment:
Thank you for pointing that out. I could not really think of a good place for this file since its used to produce cyclus input file by declaring facilities and DeployInsts.

I'll try to think of a better place to put this file.